packages/shell-api/src/mongo.ts

```diff
@@ -83,7 +84,7 @@ export default class Mongo extends ShellApiClass {
     assertArgsDefined(name);
     assertArgsType([name], ['string']);
     if (!name.trim()) {
-      throw new MongoshInvalidInputError('Database name cannot be empty.');
+      throw new MongoshInvalidInputError('Database name cannot be empty.', CommonErrors.NotImplemented);
```

Comment:
This should be `InvalidArgument` I think?
```suggestion
      throw new MongoshInvalidInputError('Database name cannot be empty.', CommonErrors.InvalidArgument);
```